app/services/organizations/delete.rb

```diff
@@ -15,5 +18,24 @@ def self.call(...)
     private
 
     attr_reader :org
+    attr_accessor :article_ids
+
+    def delete_notifications
+      sql = <<-SQL.squish
+        DELETE FROM notifications
+        WHERE notifications.id IN (
+          SELECT notifications.id
+          FROM notifications
+          WHERE organization_id = ?
+        )
+      SQL
+      notification_sql = Notification.sanitize_sql([sql, org.id])
+      BulkSqlDelete.delete_in_batches(notification_sql)
+    end
+
+    def articles_sync
+      # Syncs article cached organization and updates Elasticsearch docs
```

Comment:
We're close to merging the PR that will remove Elasticsearch, maybe we can update the comment here to remove the reference to it.

Reply:
This method can be removed, now that we don't have Elasticsearch anymore, correct?